R/pkg/DESCRIPTION

```diff
@@ -54,5 +54,5 @@ Collate:
     'types.R'
     'utils.R'
     'window.R'
-RoxygenNote: 5.0.1
+RoxygenNote: 6.0.1
```

Comment:
Yeah I was going to ask about this. I didn't change this myself, but it got changed when I ran the build (is that to be expected?) I do have a later Roxygen installed, but I don't think anything changed that would require it, but, I don't know the implications of this either. I can just revert it of course as it's not related.